flow/designs/asap7/mock-cpu/constraint.sdc

```diff
@@ -2,7 +2,8 @@
 #
 # This fifo is from http://www.sunburst-design.com/papers/CummingsSNUG2002SJ_FIFO1.pdf
 
-source designs/src/mock-array/util.tcl
+set script_dir [file dirname [info script]]
+source $script_dir/../../src/mock-array/util.tcl
```

Comment:
@oharboe Our CI is failing in asap7/mock-cpu with the following error:
` Error: 1_synth.sdc, 6 couldn't read file "./results/asap7/mock-cpu/base/../../src/mock-array/util.tcl": no such file or directory`

I believe it is related to this change. Do you have any advice on how to fix this? It seems that the `$script_dir` is not properly initialized.

Reply:
@maliberty @vvbandeira Weird. This was automerged, how did that happen? It doesn't build.

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/3112

Reply:
I guess we don't have this design in the PR pipelines.